stl/msbuild/stl_post/msvcp_post.settings.targets

```diff
@@ -52,11 +52,21 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
             Include="$(CrtRoot)\github\stl\msbuild\stl_codecvt_ids\$(CrtBuildModel)\msvcp_codecvt_ids_$(MsvcpFlavor)\msvcp_codecvt_ids.nativeproj"/>
         <ProjectReference Condition="'$(CanSkipProjectReferenceForTargetLib)' != 'true'"
             Include="$(CrtRoot)\github\stl\msbuild\stl_atomic_wait\$(CrtBuildModel)\msvcp_atomic_wait_$(MsvcpFlavor)\msvcp_atomic_wait.nativeproj"/>
-        <Lib Include="$(CrtLibPath)\msvcprt_base$(BuildSuffix)$(ClrLibSuffix).lib"/>
-        <Lib Include="$(CrtLibPath)\msvcprt_1$(BuildSuffix)$(ClrLibSuffix).lib"/>
-        <Lib Include="$(CrtLibPath)\msvcprt_2$(BuildSuffix)$(ClrLibSuffix).lib"/>
-        <Lib Include="$(CrtLibPath)\msvcprt$(BuildSuffix)_codecvt_ids$(ClrLibSuffix).lib"/>
-        <Lib Include="$(CrtLibPath)\msvcprt$(BuildSuffix)_atomic_wait$(ClrLibSuffix).lib"/>
+        <Lib Include="$(CrtLibPath)\msvcprt_base$(BuildSuffix)$(ClrLibSuffix).lib">
+            <IncludeInChameleonLib>false</IncludeInChameleonLib>
+        </Lib>
+        <Lib Include="$(CrtLibPath)\msvcprt_1$(BuildSuffix)$(ClrLibSuffix).lib">
+            <IncludeInChameleonLib>false</IncludeInChameleonLib>
+        </Lib>
+        <Lib Include="$(CrtLibPath)\msvcprt_2$(BuildSuffix)$(ClrLibSuffix).lib">
+            <IncludeInChameleonLib>false</IncludeInChameleonLib>

```

Comment:
what's a chameleon lib?

Reply:
A chameleon binary contains an ARM64 binary and an ARM64EC or x64 binary. It can be used by either emulated x64 or native arm64 processes.